### PR TITLE
terraform 1.1.0

### PR DESCRIPTION
**Pull request by jidicula**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
>   2021/12/08 18:28:12 Unable to parse Plan: Unable to read Plan: unsupported state format version: expected ["0.1" "0.2"], got "1.0"

test needs to be updated.

**Comment by Tenzer** [comment]
Isn't the problem rather that [terraform-rover](https://github.com/im2nguyen/rover) doesn't currently support Terraform 1.1.0? It's the test for that which is failing, and I don't think that can be properly fixed until it has support for the new state version.

**Comment by Tenzer** [comment]
I have made a pull request for rover to update its dependencies, which I believe is what's necessary for it to support the Terraform 1.1.0 state format: https://github.com/im2nguyen/rover/pull/65.

**Comment by chenrui333** [comment]
Thanks @Tenzer!

**Comment by wyardley** [comment]
Any word on how long it will take for this to get out?

**Comment by SMillerDev** [comment]
We want to make sure that nothing breaks based on this (and something did). But if the PR for rover hasn't been accepted next week we can just try and add it as a patch and build based on that.

File: Formula/terraform-rover.rb [hunk 1]
```diff
@@ -20,6 +21,12 @@ class TerraformRover < Formula
   depends_on "node"
   depends_on "terraform"
 
+  # Fix test, remove in next version
```
Comment (SMillerDev) [inline Formula/terraform-rover.rb:24]:
```suggestion
  # Update terraform components, remove in next version
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @jidicula @Tenzer!

**Pull request closed without merging**